Ark.Tools.ResourceWatcher/ResourceWatcherDiagnosticSource.cs

```diff
@@ -191,8 +190,8 @@ public Activity ProcessResourceStart(ProcessContext processContext)
         {
             _logger.Info("({4}/{5}) Detected change on ResourceId=\"{0}\", Resource.Modified={1}, OldState.Modified={2}, OldState.Retry={3}. Processing..."
                 , processContext.CurrentInfo.ResourceId
-                , processContext.CurrentInfo.Modified
-                , processContext.LastState?.Modified
+                , (processContext.CurrentInfo.ModifiedMultiple != null && processContext.CurrentInfo.ModifiedMultiple.Any()) ? processContext.CurrentInfo.ModifiedMultiple.Max(x => x.Value) : processContext.CurrentInfo.Modified
+                , (processContext.LastState?.ModifiedMultiple != null && processContext.LastState.ModifiedMultiple.Any()) ? processContext.LastState?.ModifiedMultiple.Max(x => x.Value) : processContext.LastState?.Modified
```

Comment:
for the same reasons as above, using MAX is not informative. We excatly want to cover the case in which the MAX doesn't change but another Modified is updated. If MAX would sufficed, a single Modified would be enough and the implementation would do the MAX (as of now).